eng/versioning/verify_release_set.ps1

```diff
@@ -8,6 +8,7 @@ param(
 )
 
 . "${PSScriptRoot}/../common/scripts/common.ps1"
+. "${PSScriptRoot}/../scripts/Language-Settings.ps1"
```

Comment:
Language settings is already pulled in by common so this shouldn't be needed.

Reply:
Good catch, removed with second commit.